src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -973,13 +990,18 @@ def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
 
 
 def load_wasting_rr(key: str, location: str) -> pd.DataFrame:
```

Comment:
If `location` can be an int, the typing of this function should be as follows:
```suggestion
def load_wasting_rr(key: str, location: Union[str, int]) -> pd.DataFrame:
```